From 3c928a3bf8d873f6173a076da6e4c1bc85e9a3a0 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Wed, 28 Jun 2017 20:41:49 +0300
Subject: [PATCH] Do not use win32 specific filehandling

PyFile_FromString and PyFile_AsFile are no longer available in python3
see https://sourceware.org/bugzilla/show_bug.cgi?id=15600

Upstream-Status: Pending
---
 gdb/python/python.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdb/python/python.c b/gdb/python/python.c
index 9bccaa9..1d58dff 100644
--- a/gdb/python/python.c
+++ b/gdb/python/python.c
@@ -374,7 +374,7 @@ python_interactive_command (char *arg, int from_tty)
 static void
 python_run_simple_file (FILE *file, const char *filename)
 {
-#ifndef _WIN32
+#ifdef _WIN32
 
   PyRun_SimpleFile (file, filename);
 
